pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -52,8 +53,14 @@
 class TPUSpawnPlugin(DDPSpawnPlugin):
```

Comment:
Should we define a TPUCheckpointIOPlugin?

Reply:
If the TrainingTypePlugin is used to move the weights on cpu before saving, I don't think this would be required.